modules/builds-configmaps-use-case-consuming-in-env-vars.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-consuming-in-env-vars_{context}"]
+= Consuming ConfigMaps in environment variables
+
+ConfigMaps can be used to populate individual environment variables or to populate environment variables from all keys that form valid environment variable names. As an example, consider the following ConfigMaps:
+
+.ConfigMap with two environment variables
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: special-config <1>
+  namespace: default
+data:
+  special.how: very <2>
+  special.type: charm <2>
+----
+<1> Name of the ConfigMap.
+<2> Environment variables to inject.
+
+.ConfigMap with one environment variable
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-config <1>
+  namespace: default
+data:
+  log_level: INFO <2>
+----
+<1> Name of the `ConfigMap`.
+<2> Environment variable to inject.
+
+.Procedure
+
+* You can consume the keys of this ConfigMap in a pod using `configMapKeyRef` sections.
```

Comment:
I found it hard to understand what `configMapKeyRef` is actually doing here. Would it be more explicit to say something like this?:

* You can inject the key values from ConfigMaps into a Pod's environment variables. The following example injects the value `very` from the `special.how` key in the `special-config` ConfigMap into a pod environment variable called `SPECIAL_LEVEL_KEY`: